### ISSUE TITLE
CT reentry detected, value=Vector2(0.75, -0.5), oldValue=Vector2(1.5, -0.25)

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/quadrilateral#303]
```
quadrilateral : phet-io-fuzz : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1671770776298/quadrilateral/quadrilateral_en.html?continuousTest=%7B%22test%22%3A%5B%22quadrilateral%22%2C%22phet-io-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1671770776298%22%2C%22timestamp%22%3A1671772725586%7D&ea&brand=phet-io&phetioStandalone&fuzz&memoryLimit=1000
Query: ea&brand=phet-io&phetioStandalone&fuzz&memoryLimit=1000
Uncaught Error: Assertion failed: reentry detected, value=Vector2(0.75, -0.5), oldValue=Vector2(1.5, -0.25)
Error: Assertion failed: reentry detected, value=Vector2(0.75, -0.5), oldValue=Vector2(1.5, -0.25)
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1671770776298/assert/js/assert.js:28:13)
at assert (ReadOnlyProperty.ts:308:14)
at _notifyListeners (ReadOnlyProperty.ts:359:46)
at deferredListener (QuadrilateralShapeModel.ts:1395:79)
at Array.forEach
at forEach (QuadrilateralShapeModel.ts:1395:30)
at setPropertiesDeferred (QuadrilateralShapeModel.ts:1123:9)
at set (QuadrilateralModel.ts:278:37)
at areVertexPositionsAllowed (SideNode.ts:284:54)
at _dragListener (PressListener.ts:526:9)
id: Bayes Puppeteer
Snapshot from 12/22/2022, 9:46:16 PM

----------------------------------

quadrilateral : phet-io-fuzz : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1671770776298/quadrilateral/quadrilateral_en.html?continuousTest=%7B%22test%22%3A%5B%22quadrilateral%22%2C%22phet-io-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1671770776298%22%2C%22timestamp%22%3A1671780451167%7D&ea&brand=phet-io&phetioStandalone&fuzz&memoryLimit=1000
Query: ea&brand=phet-io&phetioStandalone&fuzz&memoryLimit=1000
Uncaught Error: Assertion failed: reentry detected, value=Vector2(0.25, -0.25), oldValue=Vector2(0.25, 0.25)
Error: Assertion failed: reentry detected, value=Vector2(0.25, -0.25), oldValue=Vector2(0.25, 0.25)
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1671770776298/assert/js/assert.js:28:13)
at assert (ReadOnlyProperty.ts:308:14)
at _notifyListeners (ReadOnlyProperty.ts:359:46)
at deferredListener (QuadrilateralShapeModel.ts:1395:79)
at Array.forEach
at forEach (QuadrilateralShapeModel.ts:1395:30)
at setPropertiesDeferred (QuadrilateralShapeModel.ts:1123:9)
at set (QuadrilateralModel.ts:278:37)
at areVertexPositionsAllowed (SideNode.ts:284:54)
at _dragListener (PressListener.ts:526:9)
id: Bayes Puppeteer
Snapshot from 12/22/2022, 9:46:16 PM
```

**Comment by jessegreenberg** [comment]
I let the sim fuzz in phet brand for many minutes because I didn't think this would be related to phet-io. But I never saw this. phet-io fuzzing has been blocked by #312. Ill try fuzzing in phet-io to see if this shows up.

**Comment by jessegreenberg** [comment]
A different error appeared with phet-io fuzz, Ill make a new issue.

**Comment by jessegreenberg** [comment]
This will stop showing up on CT after #314.  We can come back and support PhET-iO if we need to later.

**Comment by jessegreenberg** [comment]
This showed up again on CT after fixing #312

**Comment by BLFiedler** [comment]
Is this still there for master after recent QA tests?

**Comment by jessegreenberg** [comment]
Yes I think so, but it is specific to PhET-iO. Its going to be hidden until we re-enable PhET-iO in https://github.com/phetsims/quadrilateral/issues/312.

**Comment by BLFiedler** [comment]
No intention to resume phet-io use at this time. Only for research purposes (none on immediate horizon), but not publication. Might crop back up at such a time, but closing for now.
